builtin.c

```diff
@@ -707,44 +707,7 @@ XS(XS_builtin_import)
 XS(XS_builtin_unimport);
 XS(XS_builtin_unimport)
 {
-    dXSARGS;
-
-    if(!PL_compcv)
-        Perl_croak(aTHX_
-                "builtin::unimport can only be called at compile time");
-
-    prepare_export_lexical();
-
-    for(int i = 1; i < items; i++) {
-        SV *sym = ST(i);
-        const char *sympv = SvPV_nolen(sym);
-        if(strEQ(sympv, "import") || strEQ(sympv, "unimport"))
-            Perl_croak(aTHX_ builtin_not_recognised, sym);
-
-        SV *ampname = sv_2mortal(Perl_newSVpvf(aTHX_ "&%" SVf, SVfARG(sym)));
-        SV *fqname = sv_2mortal(Perl_newSVpvf(aTHX_ "builtin::%" SVf, SVfARG(sym)));
-
-        CV *cv = get_cv(SvPV_nolen(fqname), SvUTF8(fqname) ? SVf_UTF8 : 0);
-        if(!cv)
-            Perl_croak(aTHX_ builtin_not_recognised, sym);
-
-        PADOFFSET off = pad_findmy_sv(ampname, 0);
-        if((off == NOT_IN_PAD) ||
-                (PL_curpad[off] != (SV *)cv))
-            Perl_croak(aTHX_
-                    "'%" SVf "' does not appear to be an imported builtin function", SVfARG(ampname));
-
-        /* Add a tombstone entry */
-        /* TODO: If the pad entry we found is going to go out of scope at the
-         * same time as this tombstone would, we could not bother adding the
-         * tombstone and instead COP_SEQ_MAX_HIGH_set() on the padname to
-         * clear it.
-         */
-        pad_add_name_sv(ampname, padadd_STATE|padadd_TOMBSTONE, 0, 0);
-    }
-
-    COP_SEQMAX_INC;
-    finish_export_lexical();
+    croak("builtin::unimport has now been removed");
```

Comment:
`builtin::unimport` was never in a stable release, so there's no point in leaving this tombstone here, just remove it completely.

Reply:
Indeed - I'll do that before I undraft the PR. I just wanted to see the errors during some smoke-tests first.

Reply:
done